pulseapi/users/admin.py

```diff
@@ -3,33 +3,46 @@
 """
 from django.contrib import admin
 from django.contrib.auth.models import Group
+from django.utils.html import format_html
+
 from .models import EmailUser
+from pulseapi.profiles.models import UserProfile, UserBookmarks
 
-class UserBookmarksInline(admin.TabularInline):
-    """
-    We need an inline widget before we can do anything
-    with the user/entry bookmark data.
-    """
-    model = EmailUser.bookmarks.through
-    verbose_name = 'UserBookmarks'
 
 
 class EmailUserAdmin(admin.ModelAdmin):
     """
     Show a list of entries a user has submitted in the EmailUser Admin app
     """
-    fields = ('password', 'last_login', 'email', 'name', 'entries','bookmarks', 'is_staff', 'is_superuser')
-    readonly_fields = ('entries','bookmarks')
-
-    # this allows us to create/edit/delete/etc bookmarks:
-    inlines = [ UserBookmarksInline ]
+    fields = ('password', 'last_login', 'email', 'name', 'is_staff', 'is_superuser', 'profile', 'entries','bookmarks', )
```

Comment:
Both. It's useful for admins to see the bookmark count in the user view (although I took out the inline, there is no reason to see each bookmark as editable entity...)